src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/src/upd_request_operation.c

```diff
@@ -499,7 +503,7 @@ static void HandleReportResultsAck(ADUC_Retriable_Operation_Context* context, AD
             context->attemptCount = 0;
             context->lastFailureTime = nowTime;
             context->nextExecutionTime = nowTime + 60; // TODO: add random jitter and use retry configuration.
-            AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData);
+            AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData, context);
```

Comment:
#define for the + 60. Same question about the nowTime verses using the time we transition state and/or send the report to prevent drift from caller execution to this function.